core/domain/user_jobs.py

```diff
@@ -233,6 +269,40 @@ def map(item):
                                 'type': 'exploration',
                                 'id': item.id
                             })
+        elif isinstance(item, collection_models.CollectionRightsModel):
+            if item.deleted:
```

Comment:
That's reasonable. Could you document the rationale, so that future developers understand why? I think it's non-obvious.